ERR_MODULE_NOT_FOUND on v4 and v5

I followed the repository instructions:

```
npm i -g @github/local-action

local-action --help
```

But this results in the following error:
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/redacted.nvm/versions/node/v20.19.2/lib/node_modules/@github/local-action/src/index.ts--'
```

This seems to stem from:
https://github.com/github/local-action/blob/main/bin/local-action.js#L58

Changing that line to:
```
command += ` -- --help`
``` 
(extra space)

Fixed the issue.

--- a/bin/local-action.js
+++ b/bin/local-action.js
@@ -55,7 +55,7 @@ function entrypoint() {
     // If the TARGET_ACTION_PATH environment variable is not set, display the
     // help message.
     if (!process.env.TARGET_ACTION_PATH) {
-      command += `-- --help`
+      command += ` -- --help`
       execSync(command, { stdio: 'inherit' })
       return
     }

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@github/local-action",
   "description": "Local Debugging for GitHub Actions",
-  "version": "5.0.0",
+  "version": "5.0.1",
   "type": "module",
   "author": "Nick Alteen <ncalteen@github.com>",
   "private": false,

